debezium-core/src/main/java/io/debezium/data/geometry/Point.java

```diff
@@ -14,12 +14,14 @@
  * A semantic type for a geometric Point, defined as a set of (x,y) coordinates.
  * 
  * @author Horia Chiorean
+ * @author oalsafi
  */
 public class Point {
 
     public static final String LOGICAL_NAME = "io.debezium.data.geometry.Point";
     public static final String X_FIELD = "x";
     public static final String Y_FIELD = "y";
+    public static final String WKB_FIELD = "wkb"; //Please see DBZ-208
```

Comment:
Actually I was referring to discussion points I had and I wrote them on DBZ-208, maybe I shall move discussion point related to POINT to DBZ-222?